src/games/kalambury/data/phrases/index.js

```diff
@@ -0,0 +1,32 @@
+import * as en from "./en";
+import * as pl from "./pl";
+
+const languages = { pl, en };
+
+export const languagesList = [
+  {
+    key: "pl",
+    name: "polski",
+    sets: [
+      { key: "idioms", name: "Idiomy" },
+      { key: "noun_phrases", name: "Frazy" },
```

Comment:
@glowacki-dev I'm not sure if that's a good translation 🤔

Reply:
wyrażenia?